### PR TITLE
chore: remove lodash from direct dependencies

### PR DESCRIPTION
**Pull request by rishabh3112**
**What kind of change does this PR introduce?**
chore

**Did you add tests for your changes?**
No

**If relevant, did you update the documentation?**
No

**Summary**
Remove lodash from dependencies to reduce deps

**Does this PR introduce a breaking change?**
No

**Other information**
As per feedback for v4

### CODE REVIEW

**Review by evilebottnawi — APPROVED** [review]
@rishabh3112 Do we remove lodash everywhere?

**Review by evenstensberg — APPROVED** [review]

File: yarn.lock [hunk 1]
```diff
@@ -8488,7 +8483,7 @@ lodash@4.17.15:
   resolved "https://registry.yarnpkg.com/lodash/-/lodash-4.17.15.tgz#b447f6670a0455bbfeedd11392eff330ea097548"
   integrity sha512-8xOcRHvCjnocdS5cpwXQXVzmmh5e5+saE2QGoeQmbKmRS6J3VQppPOIt0MnmE+4xlZoumy0GPG0D0MVIQbNA1A==
 
-lodash@4.17.19, lodash@^4.17.10, lodash@^4.17.11, lodash@^4.17.12, lodash@^4.17.13, lodash@^4.17.14, lodash@^4.17.15, lodash@^4.17.19, lodash@^4.17.4, lodash@^4.2.1:
+lodash@^4.17.10, lodash@^4.17.11, lodash@^4.17.12, lodash@^4.17.13, lodash@^4.17.14, lodash@^4.17.15, lodash@^4.17.19, lodash@^4.17.4, lodash@^4.2.1:
```
Comment (anshumanv) [inline yarn.lock:8486]: it seems there are a bunch of lodash versions via transitive dep 😞

**Review by anshumanv — APPROVED** [review]

File: yarn.lock (hunk 1 above)
Comment (evilebottnawi) [reply yarn.lock:8486]: Don't worry, we should reduce our size :smile:

**Review by snitin315 — APPROVED** [review]

**Comment by rishabh3112** [comment]
> @rishabh3112 Do we remove lodash everywhere?

I have removed lodash where ever it was direct dep. Still many versions of lodash are shipped as transitive dep as mentioned by @anshumanv in his review as well.

File: yarn.lock (hunk 1 above)
Comment (rishabh3112) [reply yarn.lock:8486]: Do you have any idea for this?

**Comment by evilebottnawi** [comment]
Feel free to merge after green

**Comment by anshumanv** [comment]
> I have removed lodash where ever it was direct dep. Still many versions of lodash are shipped as transitive dep as mentioned by @anshumanv in his review as well.

We should remove version locking in packages, maybe that will reduce some

**Comment by anshumanv** [comment]
Will try to do it this weekend

**Comment by evilebottnawi** [comment]
@anshumanv 

> We should remove version locking in packages, maybe that will reduce some

Where we have it?

**Comment by anshumanv** [comment]
@evilebottnawi https://github.com/webpack/webpack-cli/pull/958

**Comment by evilebottnawi** [comment]
@webpack/cli-team please focus on use `^` everywhere in `package.json` in the next PR, it is important

**Pull request merged**